src/github.com/getlantern/flashlight/logging/logging.go

```diff
@@ -90,10 +93,14 @@ func Configure(addr string, cloudConfigCA string, instanceId string,
 
 	// Using a goroutine because we'll be using waitforserver and at this time
 	// the proxy is not yet ready.
+	done = make(chan bool, 1)
```

Comment:
I don't understand this change. You're making the last few lines of this non-blocking by default unless the caller chooses to block with the returned channel. The previous code blocked, and your new code also chooses to use the returned channel to block. So the effect of this change is that the [previous call](https://github.com/getlantern/lantern/blob/issue-2648/src/github.com/getlantern/flashlight/flashlight.go#L356) to this does not block, so Loggly will not be configured when it returns. I'm not sure what the effect of that will be, but presumably some future logging calls may not yet have Loggly configured. 

What does this buy us? Configuring Loggly costs basically nothing, so I don't understand why we'd want it to be configured asynchronously.